lib/Model.php

```diff
@@ -571,7 +572,7 @@ public function attributes()
 	/**
 	 * Retrieve the primary key name.
 	 *
-	 * @param boolean Set to true to return the first value in the pk array only
+	 * @param boolean $first Set to true to return the first value in the pk array only
```

Comment:
Not saying that, but this PR seems pretty inconsistent where periods are added and other places they are not.